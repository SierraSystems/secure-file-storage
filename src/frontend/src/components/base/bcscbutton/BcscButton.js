import React from "react";
import PropTypes from "prop-types";
import "./BcscButton.css";

export const BcscButton = ({ onClick }) => (
  <div className="outline">
    <div className="center-align">
      <button className="login_bcsc btn" onClick={onClick} type="button">
        Login with a BC Services Card
      </button>
    </div>
  </div>
);

BcscButton.propTypes = {
  onClick: PropTypes.func.isRequired
};
