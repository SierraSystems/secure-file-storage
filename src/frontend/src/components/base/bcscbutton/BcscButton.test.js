import React from "react";
import { BcscButton } from "./BcscButton";
import testBasicSnapshot from "../../../TestHelper";

const onClick = jest.fn();

describe("BcscButton", () => {
  test("Component renders as expected", () => {
    testBasicSnapshot(<BcscButton onClick={onClick} />);
  });
});
