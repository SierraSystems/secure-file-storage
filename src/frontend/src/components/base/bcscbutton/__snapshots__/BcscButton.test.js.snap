// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BcscButton Component renders as expected 1`] = `
<div
  className="outline"
>
  <div
    className="center-align"
  >
    <button
      className="login_bcsc btn"
      onClick={[MockFunction]}
      type="button"
    >
      Login with a BC Services Card
    </button>
  </div>
</div>
`;
